debezium-embedded/src/test/java/io/debezium/embedded/async/AsyncEmbeddedEngineTest.java

```diff
@@ -612,15 +681,15 @@ protected void waitForEngineToStart() {
         Awaitility.await()
                 .alias("Engine haven't started on time")
                 .pollInterval(TEST_TASK_MANAGEMENT_TIMEOUT_MS, TimeUnit.MILLISECONDS)
-                .atMost(1, TimeUnit.SECONDS)
+                .atMost(5 * TEST_TASK_MANAGEMENT_TIMEOUT_MS, TimeUnit.SECONDS)
```

Comment:
@vjuranek  We have {{debezium.test.records.waittime}} sys property. Maybe we can intorduce a new one or re-use it?
I can imagine that different connector will have different timout needs.

Reply:
good idea, added `debezium. test.engine.waittime`